utoipa-gen/src/component/schema.rs

```diff
@@ -588,21 +580,15 @@ impl ToTokens for UnnamedStructSchema<'_> {
             if let Some(ref attrs) = self.features {
                 tokens.extend(attrs.to_token_stream())
             }
-        };
-
-        let description = CommentAttributes::from_attributes(self.attributes).as_formatted_string();
-        if !description.is_empty() && !is_object {
-            tokens.extend(quote! {
-                .description(Some(#description))
-            })
         }
```

Comment:
The reason why there is `!is_object` check is that description cannot be set on `$ref` type.

Reply:
Yes :) But when we convert to an array we no longer have a ref. Without this change 2 of the tests I modified will not produce descriptions. 

You're right that it will error on the Ref type if we try to add a description via token extension to any unnamed struct parse. The enum reference tests catch that case and error (and do not error in this case because we're converting to array).

TLDR Ref types don't error with snippet and the snippet enables descriptions on unnamed structs with more than one field 

How would you like me to modify it?

Reply:
I'd make sure that this scenario passes:
```rust
#[derive(ToSchema)]
struct Bar {
  value: String
}

/// Description now will be tried to pass on $ref that is created for inner type Bar
#[derive(ToSchema)]
struct Foo(Bar);
```

Though this will be possible in openapi 3.1. But that needs other changes as well and is another topic for later PRs.

Reply:
Thank you. I added the above test, which doesn't fail and doesn't add a description to the ref if I understand the testing correctly.

Reply:
Great